### PR TITLE
added bitv 1.2

### PR DESCRIPTION
**Pull request by backtracking**
this is a new release of bitv

### CODE REVIEW

File: packages/bitv/bitv.1.2/opam [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  ["autoconf"]
+  ["./configure"]
+  [make]
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "bitv"]
+depends: [
+  "ocamlfind"
+  "conf-autoconf" {build}
+]
```
Comment (avsm) [inline packages/bitv/bitv.1.2/opam]: Needs `available: [ocaml-version >= "4.00.1"]` due to use of `iteri`.

On OCaml 3.12.1: 
```
# File "bitv_string.ml", line 127, characters 2-13:
# Error: Unbound value Bytes.iteri
# make: *** [bitv_string.cmo] Error 2
```

**Comment by UnixJunkie** [comment]
Superseded by https://github.com/ocaml/opam-repository/pull/8511

**Comment by UnixJunkie** [comment]
I need bitv.1.2 to be available in opam...

**Comment by UnixJunkie** [comment]
this can be closed because the superseding pull request has been accepted

**Pull request closed without merging**